lib/swoosh/adapters/smtp/helpers.ex

```diff
@@ -37,7 +37,7 @@ defmodule Swoosh.Adapters.SMTP.Helpers do
     |> prepare_from(email)
   end
 
-  defp prepare_subject(headers, %{subject: subject}), do: [{"Subject", subject} | headers]
+  defp prepare_subject(headers, %{subject: subject}), do: [{"Subject", subject || ""} | headers]
```

Comment:
This could hide bugs.
I'd prefer something like
```suggest
defp prepare_subject(headers, %{subject: subject}) when is_binary(subject), do: [{"Subject", subject} | headers]
```
and have it raise on wrong types.
What do you think?